qiskit_machine_learning/utils/loss_functions/loss.py

```diff
@@ -0,0 +1,111 @@
+from abc import ABC, abstractmethod
+import numpy as np
+
+from ...exceptions import QiskitMachineLearningError
+
+
+class Loss(ABC):
```

Comment:
Do we need a whole package for one file. If these were all in separate files I could see it. But with everything in one file I would have a thought a loss_functions.py file directly in utils would have sufficed. Unless there is some view to future expansion that would see more coming to this loss_funtions folder.

Reply:
Yes. it is our plan to add more loss in this folder.